docs/4.1. Behaviour - Registration.md

```diff
@@ -57,11 +57,11 @@ A Node SHOULD register resources in the following order to avoid rejections due
 
 (\*) Version v1.0 Flows do not have a device_id and SHOULD be garbage collected based on their parent source_id.
```

Comment:
I can't make a suggestion for the lines above this which mention `node_id`, etc.

```suggestion
(\*) Version v1.0 Flows do not have a `device_id` and SHOULD be garbage collected based on their parent `source_id`.
```

Also, somewhere above, it says "version 1.0", whereas here it says "Version v1.0". Which is right?